tools/oeutil/host/generate_evidence.cpp

```diff
@@ -91,12 +91,19 @@ static const oe_uuid_t _sgx_epid_unlinkable_uuid = {
     OE_FORMAT_UUID_SGX_EPID_UNLINKABLE};
 
 #ifdef OEUTIL_TCB_ALLOW_ANY_ROOT_KEY
-// Override weak variable in common/sgx/tcbinfo.c
+// Allow oeutil to override the root key used in
+// common/sgx/tcbinfo.c:get_trusted_root_key_pem
+
```

Comment:
Is the value below always the same?

Reply:
No, oeutil could replace it with content from a user provided file